backend.py

```diff
@@ -7,7 +7,10 @@
 from flask_limiter.util import get_remote_address
 from flask_pydantic import validate
 from pydantic.main import BaseModel
-from radis.db import MOLECULES_LIST_EQUILIBRIUM, MOLECULES_LIST_NONEQUILIBRIUM
+if radis.get_version()>'0.9.26':  # latest dev version
+	from radis.db import MOLECULES_LIST_EQUILIBRIUM, MOLECULES_LIST_NONEQUILIBRIUM
+else:
+	from radis.io import MOLECULES_LIST_EQUILIBRIUM, MOLECULES_LIST_NONEQUILIBRIUM
```

Comment:
Hm, I think this shouldn't be necessary as there is only one RADIS version pinned for this service. Couldn't we just support the latest version for right now?

Reply:
The import changed with the isort-refactor in RADIS (https://github.com/radis/radis/pull/140). I thought it was in 0.9.26-master but it is only on the develop branch, so will be in 0.9.27. The "if" will catch any radis develop version.

Reply:
@suzil if you're okay with the change I just have to lint/black it !

Reply:
I'm just a bit confused still.  radis-ui only needs to support one version. If there's a new release of RADIS, the import could just change to that instead of supporting both versions in this repo.

Reply:
It's because the change happened only on the developer version, not released yet. I understand your point so let's keep it clean : I'll update only when 0.9.27 is released!